packages/styles/scss/components/feature-card/_feature-card.scss

```diff
@@ -51,7 +51,7 @@
 
       &__heading {
         margin-bottom: $layout-03;
-        @include carbon--type-style(expressive-heading-02, true);
+        @include carbon--type-style(expressive-heading-03, true);
```

Comment:
Could we just delete this line since `Card` already uses `heading-03`?

https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles/scss/components/card/index.scss#L31